packages/database-mongo/src/mongo.ts

```diff
@@ -364,6 +364,18 @@ export class Mongo implements DatabaseInterface {
     );
   }
 
+  public async removeAllPasswordResetTokens(userId: string): Promise<void> {
+    const id = this.options.convertUserIdToMongoObjectId ? toMongoID(userId) : userId;
+    await this.collection.update(
+      { _id: id },
+      {
+        $set: {
```

Comment:
I am not sure it really makes sense to set an empty array here, we could just unset the value to leave a clean object in the db, what do you think?